Samples/PublicSamples/RecordingBot/src/RecordingBot.Services/ServiceSetup/AppHost.cs

```diff
@@ -105,6 +115,15 @@ public void Boot()
                 _logger.Error(e, "Unhandled exception in Boot()");
             }
 
+            // Configure the HTTP request pipeline.
+            app.UsePathBase(azureSettings.PodPathBase); 
+            app.UseHttpsRedirection();
+
+            app.UseAuthorization();
+
+            app.MapControllers();
+
+            app.Run();
```

Comment:
Either remove the StartServer method and move the bot service Initialize before the app.run as it blocks further execution or move the bot Service Initialization to the boot method